schema/README.md

```diff
@@ -0,0 +1,31 @@
+The `schema` module contains packages that help to parse and validate
+[schema files](https://github.com/open-telemetry/oteps/blob/main/text/0152-telemetry-schemas.md).
+
+Each `major.minor` schema file format version is implemented as a separate package, with
+the name of the package in the `vmajor.minor` form.
+
+To parse a schema file, first decide what file format version you want to parse,
+then import the corresponding package and use the `Parse` or `ParseFile` functions
+like this:
+
+```go
+import schema "go.opentelemetry.io/otel/schema/v1.0"
+
+// Load the schema from a file in v1.0.x file format.
+func loadSchemaFromFile() error {
+telSchema, err := schema.ParseFile("schema-file.yaml")
+    if err != nil {
+       return err	
+    }
+    // Use telSchema struct here.
+}
+
+// Alternatively use schema.Parse to read the schema file from io.Reader.
+func loadSchemaFromReader(r io.Reader) error {
+telSchema, err := schema.Parse(r)
+    if err != nil {
+        return err
+    }
+    // Use telSchema struct here.
```

Comment:
```suggestion
	telSchema, err := schema.ParseFile("schema-file.yaml")
	if err != nil {
		return err	
	}
	// Use telSchema struct here.
}

// Alternatively use schema.Parse to read the schema file from io.Reader.
func loadSchemaFromReader(r io.Reader) error {
	telSchema, err := schema.Parse(r)
	if err != nil {
		return err
	}
	// Use telSchema struct here.
```

Reply:
Github diff doesn't make it clear what the suggestion is. Can you clarify?

Reply:
I see now, the first line has incorrect indentation. Fixed.

Reply:
Looks like you resolved the indentation issue. Feel free to resolve if you don't want to change the spaces.

Reply:
Fixed all now. Tabs are tabs.